config/routes.rb

```diff
@@ -19,6 +17,12 @@
 
   devise_for :users
```

Comment:
Perfect!!!
Please only set this awesome solution on the top. Root is the most used url and its good to set it at top.
```
devise_for :users

  authenticated :user do
    root 'pages#dashboard', as: :authenticated_root
  end

root 'pages#home'
```